index.html

```diff
@@ -29,13 +29,8 @@
                         "Please enter a name for the new document:",
                     )
                     var newDoc = {id: 42, name: name, content: ""} // FIXME
-                    setActive(newDoc.id)
```

Comment:
This needs to come after `setDocuments()` because it otherwise references something that doesn't exist in `documents` yet.